src/test/java/com/uvocab/api/service/AuthServiceTest.java

```diff
@@ -0,0 +1,26 @@
+package com.uvocab.api.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+
+public class AuthServiceTest {
```

Comment:
@Danilosimoes see as attention  this Unit test, the **AuthService** have a **dependecie** and I not see the mock configuration for that.